src/stripe/stripe-service.ts

```diff
@@ -186,6 +192,10 @@ export class StripeService {
     });
   }
 
+  static createAddressElement(elements: StripeElements) {
+    return elements.create("address", { mode: "billing" });
+  }
```

Comment:
We could improve the customer experience by passing tax customer details as default values:
```suggestion
  static createAddressElement(
    elements: StripeElements,
    country: string,
    postalCode?: string,
  ) {
    return elements.create("address", {
      mode: "billing",
      defaultValues: {
        address: {
          country: country,
          postal_code: postalCode,
        },
      },
    });
  }
```